querybook/webapp/redux/comment/reducer.ts

```diff
@@ -30,13 +30,19 @@ function commentReducer(state = initialState, action: CommentAction) {
                 draft.commentsById[parentCommentId].child_comment_ids.push(
                     childCommentId
                 );
-
+                return;
+            }
+            case '@@comment/ARCHIVE_COMMENT': {
+                const { commentId } = action.payload;
+                draft.commentsById[commentId] = {
```

Comment:
draft.commentsById[commentId].archived = True

no need to spread since you are using Immer